mne/viz/evoked.py

```diff
@@ -311,10 +311,18 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                         for i in bad_ch_idx:
                             if i in idx:
                                 colors[idx.index(i)] = 'r'
-                    for ch_idx in range(len(D)):
+
+                    # find the channels with the least activity,
+                    # and map them in front of the more active ones
+                    auc = np.trapz((np.abs(D - D.mean(1).reshape(-1, 1))))
```

Comment:
shouldn't it `abs(D)`? Everything has to be rectified, else e.g. negative potential can interact with later positive potentials